datastore/src/main/java/com/openlattice/datastore/data/controllers/DataController.java

```diff
@@ -756,14 +787,7 @@ public Long deleteEntitiesAndNeighbors(
 
         ensureEntitySetCanBeWritten( entitySetId );
 
-        WriteEvent writeEvent = deletionManager.clearOrDeleteEntitiesAndNeighborsIfAuthorized(
-                entitySetId,
-                filter.getEntityKeyIds(),
-                filter.getSrcEntitySetIds().orElse( ImmutableSet.of() ),
-                filter.getDstEntitySetIds().orElse( ImmutableSet.of() ),
-                deleteType,
-                Principals.getCurrentPrincipals()
-        );
+        WriteEvent writeEvent = new WriteEvent( 0, 0 );
```

Comment:
From what I can tell, `deleteEntitiesAndNeighbors` no longer does anything. Is this intentional?

Reply:
yes